certora/harnesses/GuardianStorageHarness.sol

```diff
@@ -0,0 +1,24 @@
+// SPDX-License-Identifier: GPL-3.0
+pragma solidity >=0.8.12 <0.9.0;
+import {GuardianStorage} from "../../contracts/modules/social_recovery/storage/GuardianStorage.sol";
+
+contract GuardianStorageHarness is GuardianStorage {
+
+    /**
+     * @dev Gets the count of guaridans for a wallet.
```

Comment:
I would describe how this is different than the `guardiansCount` storage that already exists (i.e. you are counting them in a linked list) and more importantly _why_ this is needed.

Reply:
Done 👍🏾